osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -430,24 +431,20 @@ private void refreshBPMLabel()
                 ForceRedraw();
             }
 
-            private OsuSpriteText[] getMapper(BeatmapMetadata metadata)
+            private LinkFlowContainer getMapper(BeatmapMetadata metadata)
             {
-                if (string.IsNullOrEmpty(metadata.Author?.Username))
-                    return Array.Empty<OsuSpriteText>();
+                if (metadata.Author == null)
+                    return new LinkFlowContainer();
```

Comment:
```suggestion
                    return Empty();
```

\+ change return type of method to `Drawable`